packages/firestore-types/index.d.ts

```diff
@@ -37,7 +37,7 @@ export interface PersistenceSettings {
   experimentalTabSynchronization?: boolean;
 }
 
-export type LogLevel = 'debug' | 'error' | 'silent';
+export type LogLevel = 'debug' | 'error' | 'silent' | 'warn';
```

Comment:
Do we not want to expose all levels? You are missing `verbose` and `info`.

Reply:
Done.